### PR TITLE
Fix activate no cookie bug

### LINKED ISSUE #270: activate user bug
We can activate a user but they get an error message. Activation does happen as they can log in the next step.

### PR DESCRIPTION
**Pull request by JMurphyWeb**
resolves #270

### CODE REVIEW

**Comment by nelsonic** [comment]
@JMurphyWeb please `git merge master` into this branch. (_the `diff` should be pretty small after that... thanks!_)

**Comment by JMurphyWeb** [comment]
@nelsonic it is already up to date

File: server/email-templates/welcome.html [hunk 1]
```diff
@@ -1,8 +1,8 @@
 <p>Hi {{first_name}} {{last_name}}</p>
 
-<p>Welcome to CE100! You've been added as a user of {{organisation_name}}.</p>
-<p>Click <a href="localhost:3000/people/activate/{{hashedId}}">here</a> to log in.</p>
+<p>Welcome to CE100!</p>
+<p>Click <a href="https://staging-ce100-emf.herokuapp.com/people/activate/{{hashedId}}">here</a> to activate your account.</p>
```
Comment (nelsonic) [inline server/email-templates/welcome.html]: avoid _hard-coding_ values for urls in the _markup_ ... (_please add an issue to make this a handlebars property..._)

File: server/handlers/create-new-user.js [hunk 2]
```diff
@@ -12,24 +12,33 @@ module.exports = (request, reply) => {
...
+      }
     });
   });
 };
+
+function sendActivationEmail(id, user, organisation_name, callback) {
+  Iron.seal(id, process.env.COOKIE_PASSWORD, Iron.defaults, (error, hashedId) => {
+    Hoek.assert(!error, 'Iron error');
+    var newUser = organisation_name
+      ? Object.assign({}, user, { organisation_name }, { hashedId })
```
Comment (nelsonic) [inline server/handlers/create-new-user.js]: this "_works_" but it _could_ be simplified. 😉

**Comment by nelsonic** [comment]
@JMurphyWeb looks good. 👍 
jungle book

**Pull request merged**